src/components/Navbar/NavBar.tsx

```diff
@@ -146,17 +117,65 @@ const NavBar = (props: Props) => {
             i18nText="PRESS"
             altText="PRESS"
           />
-          <LanguageContainer compact={hamburgerOpen.toString()}>
-            <LanguageButton onClick={(e) => changeLanguage(e, 'en')}>
-              ENG
+        </NavLinksContainer>
+      );
+  };
+  return (
+    <HeaderContainer compact={hamburgerOpen.toString()}>
+      <Logo />
+      {hamburgerOpen ? (
+        showCompactMenu()
+      ) : (
+          <NavLinksContainer compact={hamburgerOpen.toString()}>
+            <NavLink
+              compact={hamburgerOpen.toString()}
+              href="https://sendchinatownlove.com/"
+              i18nText="navBar.header.home"
+              altText="HOME"
+            />
+            <ReactNavLink
+              to="#"
+              onClick={handleDropdownOpen}
+              compact={hamburgerOpen.toString()}
+              active={isMerchantsPathActive.toString()}
+            >
+              {t('navBar.header.merchants')}
+            </ReactNavLink>
+            {DropdownOpen && <Dropdown >
+              <DropdownItem href='https://merchant.sendchinatownlove.com/all'>{t('navBar.header.waystodonate.donate')}</DropdownItem>
```

Comment:
Anything with `merchant.*` can be routed via a ReactRouter vs redirecting and reloading the entire page. If we redirect like this, we are forgoing the real benefit of using React, which is to only reload the elements in the DOM that have changed. If you reload our SPA with a redirect, it loads everything from scratch